examples/advanced.ts

```diff
@@ -19,6 +19,7 @@ const cacheValue = 'value';
 // you can customize your log level or provide your own logger factory to
 // integrate with your favorite logging framework
 const loggerFactory: MomentoLoggerFactory = new DefaultMomentoLoggerFactory();
+const logger = loggerFactory.getLogger(`AdvancedExample`);
```

Comment:
GitHub complains about using backticks when not interpolating

```suggestion
const logger = loggerFactory.getLogger('AdvancedExample');
```